edx_proctoring/api.py

```diff
@@ -792,6 +794,8 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
     )
     if add_start_time:
         exam_attempt_obj.started_at = datetime.now(pytz.UTC)
+    elif alias_timeout:
```

Comment:
It was added originally over here: https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/api.py#L733

It seems to be referring to the fact that it's aliasing the timeout state to the completed state.